libs/model-assessment/src/lib/ModelAssessmentDashboard/utils/getModelTypeFromProps.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  DatasetTaskType,
+  getModelTypeFromExplanation,
+  getModelTypeFromTextExplanation,
+  ModelTypes
+} from "@responsible-ai/core-ui";
+
+import { IModelAssessmentDashboardProps } from "../ModelAssessmentDashboardProps";
+
+export function getModelTypeFromProps(
+  props: IModelAssessmentDashboardProps
+): ModelTypes {
+  let modelType: ModelTypes = ModelTypes.Multiclass;
+  const classNames = props.dataset.class_names;
+  if (props.dataset.task_type === DatasetTaskType.Regression) {
```

Comment:
since we have many if conditions in the function, can we do early return in each if condition? Or maybe use switch?